spec/models/course_spec.rb

```diff
@@ -434,4 +434,103 @@
       its(:content_status) { should eq(:published_with_unpublished_changes) }
     end
   end
+
+  describe "#publish_enrichments" do
+    context 'on a course with only a draft enrichment' do
+      let(:course) do
+        create(:course,
+            changed_at: 10.minutes.ago,
+          with_enrichments: [
+          [:initial_draft, created_at: 1.day.ago, updated_at: 20.minutes.ago],
+        ])
+      end
+      let(:user) { create(:user) }
+
+      it 'should publish the draft' do
+        course.publish_enrichment(user.id)
+        expect(course.enrichments.first.status).to eq 'published'
```

Comment:
Not saying this has to change, but you could also:

```
         expect(course.enrichments.first).to be_published
```

Reply:
Could also throw the `reload` in there: `expect(course.reload.enrichments.first).to be_published` and get rid of the extra line.

Reply:
reload inlining done, thanks

Reply:
to be_published done